### ISSUE TITLE
Sample vs Readme examples

### ISSUE THREAD

**Issue by roklenardic** [djanosik/Moon.OData#2]
Hi Dusan,

your sample contains the following example:
[HttpGet("")]
        public IActionResult Index(ODataOptions<Entity> options)
        {
            return View(new ODataSqlQuery(
                "SELECT FROM Entities WHERE OwnerId = @p0",
                10456, options
            ));
        }

but the readme contains this which I cannot figure out how to make it work:

[HttpGet("api/entities")]
public Task<IOData<Entity>> GetEntities(ODataOptions<Entity> options)
{
    // Do whatever you want with OData query options, 
    // build custom Micro ORM integration, etc.

    var query = new ODataSqlQuery(
        "SELECT FROM Entities WHERE OwnerId = @p0",
        10456, options
    );
}

How should I convert the query into the required Task result?

Thanks!
Rok

**Comment by djanosik** [comment]
I don't understand your problem. The `ODataSqlQuery` is here to generate SQL command form OData options. The `Task` result is not required at all.

**Comment by roklenardic** [comment]
Ok, but the 

public Task> GetEntities(ODataOptions options)

has to return something, currently it only defines a local query variable, doesn't it?

**Comment by djanosik** [comment]
Of course. You need to use the query somehow (retrieve data from database, etc.). I've created a [gist](https://gist.github.com/djanosik/8132e4a23614bf5f7cba00fd42491e32) to show you how to integrate OData SQL with MicroLite ORM, but you can use any other library (ADO.NET, Dapper, etc.).

_(The code is probably not working, but you should get the idea.)_
